lib/blocks.js

```diff
@@ -237,7 +237,7 @@ async function _setRequiredBlockHeight({ledgerNode, witnesses, blockHeight}) {
   // FIXME: make this a covered query
   const {collection} = ledgerNode.storage.events;
   const result = await collection.updateMany({
-    'meta.continuity2017.creator': {$in: witnesses},
+    'meta.continuity2017.creator': {$in: Array.from(witnesses)},
```

Comment:
It seems awkward for a storage API to convert a Set to an Array? My preference would be to revert this change. I understand the convenience of using a Set instead, so this is not a required fix.

Reply:
It should just be an array in the database, this item about using sets was supposed to be a minimal internal API change.

Reply:
Yeah, so this function will be moving to the database module and it should take an array for `witnesses`. Any `Set` stuff should be internal to this module.

Reply:
this was addressed here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/248/commits/3512f5115a4a2649b4bf07b905e396332f02e271

please note that:

```
andrewjones@Andrews-MacBook-Pro-2 bedrock-ledger-consensus-continuity % grep -inr _setRequiredBlockHeight ./lib 
./lib/blocks.js:154:    _setRequiredBlockHeight(
./lib/blocks.js:247:async function _setRequiredBlockHeight({ledgerNode, witnesses, blockHeight}) {
./lib/peerEvents.js:1043:      _setRequiredBlockHeight(
./lib/peerEvents.js:1060:        _setRequiredBlockHeight(
./lib/peerEvents.js:1129:    _setRequiredBlockHeight({eventRecord, blockHeight, eventMap});
./lib/peerEvents.js:1233:function _setRequiredBlockHeight({eventRecord, blockHeight, eventMap}) {
```

`_setRequiredBlockHeight` in `./lib/peerEvents.js` is a different function than `_setRequiredBlockHeight` in `./lib/blocks.js`.